django_rq/views.py

```diff
@@ -58,6 +60,12 @@ def stats(request):
     return render(request, 'django_rq/stats.html', context_data)
 
 
+def _get_all_workers(config, workers_collections):
+    for collection in workers_collections:
```

Comment:
Don't forget to call `filter_connection_params` here, or it won't work if you have connections with `DEFAULT_TIMEOUT`.